core/samples/metrics/main.cpp

```diff
@@ -0,0 +1,75 @@
+/// [Include metrics]
+#include <cubos/core/metrics.hpp>
+/// [Include metrics]
+#include <chrono>
+#include <thread>
```

Comment:
I dont think you need those includes

Reply:
I do: `std::this_thread::sleep_for(std::chrono::seconds(2));`

Reply:
at least, the `thread` one, `chrono` can be removed

Reply:
Ah right, mb